oppia/oppia-android
# Fix #1560: Chapter summary support added correctly

## Explanation


Fixes #1560 

Currently in develop, we don't have any support for chapter `outline/summary`. This PR makes sure that we display chapter summary correctly.

## Before
![Screenshot_1596711420](https://user-images.githubusercontent.com/9396084/89524145-abf8d600-d801-11ea-8736-bde34611e3bc.png)

## After
![Screenshot_1596710907](https://user-images.githubusercontent.com/9396084/89524162-b3b87a80-d801-11ea-9d9b-dedf964a9d6c.png)
![Screenshot_1596710917](https://user-images.githubusercontent.com/9396084/89524168-b7e49800-d801-11ea-8b26-3f5cc7f87eb0.png)

## How to test:
1. Run app, Fractions, go to StoryActivity, notice that we display  chapter summary rich text.
2. Run test cases in this PR.

## Checklist
